### ISSUE TITLE
Enhancement: $link attribute for "database" channel

### ISSUE THREAD

**Issue by gitrequests** [tuyakhov/yii2-notifications#16]
Add support for URL storing for notifications in database. I think it is good feature for this awesome extension. Need to update Notification Model class.

Right now the solution is parsing $body attribute for storing url, for example:
```PHP
$notification->body = "/some/url||message";

...

    $urlMessage = explode('||', $notification->body, 2);
    $url = count($urlMessage) == 2 ? $urlMessage[0] : null;
    $message = count($urlMessage) == 2 ? $urlMessage[1] : $urlMessage[0];
```

**Comment by tuyakhov** [comment]
@gitrequests Thanks for reporting this.
A notification could have a lot of different attributes associated with it (URLs, related actors, icons etc.) that you might want to use in your application, therefore, it is really hard to cover all the possible variations within an extension and keep it lightweight at the same time.
I am planning to add `data` attribute that will contain JSON encoded data related to the notification in the future release major release.
However, you can already solve this problem by json encoding necessary data in `$body` attribute. It is similar to what you already doing except for using `Json::decode` instead of `explode`.
```
$message->body = Json::encode(['actionUrl' => '/some/url', 'message' => 'Test messaage']);
...
$data = Json::decode($notification->body);
$url = $data['actionUrl'];
$message = $data['message'];
```

**Comment by tuyakhov** [comment]
Resolved in #23
